### PR TITLE
Minor content/style changes to temp site

### PR DESCRIPTION
**Pull request by OTCompa**
Ticket: NA
- Adds Tailwind support to pages in `pages/`
- Update formatting/styles for index page
- Change FRU dropdown to link nowhere (for now)
- Minor clarification updates for UCoB, DSR, TOP

### CODE REVIEW

**Review by clairefleuret — APPROVED** [review]
LGTM. As a reminder, this doesn't need to be perfect and is LITERALLY just a move from what we have in the discord to a site.

**Review by clairefleuret — CHANGES_REQUESTED** [review]
image
Can we remove this page @OTCompa

File: naurffxiv/src/pages/_app.jsx [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export default function App({ Component, pageProps }) {
-    return <Component {...pageProps} />
-  }
\ No newline at end of file
+import "../styles/globals.css";
+export default function MyApp({ Component, pageProps }) {
+return <Component {...pageProps} />;}
```
Comment (brtran4) [inline naurffxiv/src/pages/_app.jsx:3]: nit: tabbing issues

File: naurffxiv/src/pages/ultimates/_meta.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export default {
   fru: {
+    display: 'hidden'
```
Comment (brtran4) [inline naurffxiv/src/pages/ultimates/_meta.js:3]: the latest build still has it shown. not sure if that's intended

**Review by brtran4 — CHANGES_REQUESTED** [review]

File: naurffxiv/src/pages/ultimates/_meta.js (hunk 2 above)
Comment (OTCompa) [reply naurffxiv/src/pages/ultimates/_meta.js:3]: This only hides it from the sidebar (as claire previously requested)
I'll remove it from the dropdown

**Review by brtran4 — APPROVED** [review]

**Pull request merged**